src/doc/src/reference/manifest.md

```diff
@@ -809,9 +809,9 @@ crate-type = ["dylib"] # could be `staticlib` as well
 ```
 
 The available options are `dylib`, `rlib`, `staticlib`, `cdylib`, and
-`proc-macro`. You should only use this option in a package. Cargo will always
-compile packages (dependencies) based on the requirements of the package that
-includes them.
+`proc-macro`. You should only use this option in a package. Be sure to include
+`rlib` as one of the crate types for a crate intended to be used as a
```

Comment:
This doesn't seem to really make things clearer for me.  dylib and proc-macro can be used as dependencies, too.  

I would just remove the sentence, I don't feel like there's much to be said here in a short space.  Best would be a guide somewhere on when and how to use the different crate types.

I would also remove the previous sentence.  "You should only use this option in a package."  Isn't everything a package? I don't know what it is trying to say.

Reply:
> I would also remove the previous sentence. "You should only use this option in a package." Isn't everything a package? I don't know what it is trying to say.

Me either, but I assumed it made sense to those more knowledgeable than me.

I'll tear it all down.